### DISCUSSION TITLE
How to share drivers, and accept the anvitation

### DISCUSSION THREAD

**Discussion by syt198945** [timdorr/tesla-api#679] [Q&A] [answered]
How to share drivers, and accept the anvitation
I see some api in json below, but I don't know how to use thest
 ```
"FETCH_VEHICLE_SHARED_DRIVERS": {
    "TYPE": "GET",
    "URI": "api/1/vehicles/{vehicle_id}/drivers",
    "AUTH": true
  },
  "CREATE_VEHICLE_SHARE_INVITE": {
    "TYPE": "POST",
    "URI": "api/1/vehicles/{vehicle_id}/invitations",
    "AUTH": true
  },
  "FETCH_VEHICLE_SHARE_INVITES": {
    "TYPE": "GET",
    "URI": "api/1/vehicles/{vehicle_id}/invitations",
    "AUTH": true
  },
  "REVOKE_VEHICLE_SHARE_INVITE": {
    "TYPE": "POST",
    "URI": "api/1/vehicles/{vehicle_id}/invitations/{invite_id}/revoke",
    "AUTH": true
  },
  "REMOVE_VEHICLE_SHARE_DRIVER": {
    "TYPE": "DELETE",
    "URI": "api/1/vehicles/{vehicle_id}/drivers/{share_user_id}",
    "AUTH": true
  },
  "REDEEM_VEHICLE_SHARE_INVITE": {
    "TYPE": "POST",
    "URI": "api/1/invitations/redeem",
    "AUTH": true
```

**Comment by core-hacked** [comment, marked as answer]
Well, it involves some guess work, it has not yet been documented. You can try to just request the endpoint and see the response. Most of the time it gives you a "Missing paramter `name`" error then you add it and so on and so forth.

 If you do decide to do this, you could as well document it or leave a comment with what to do.

Cheers!

**Reply by syt198945** [reply]
I have tried the api  request,but i don't know the vehicle_id from the share link. Although I tried to input the right vehicle_id, the response is not right .
`{
    "response": null,
    "error": "not_found",
    "error_description": ""
}`

**Reply by core-hacked** [reply]
Wdym the right vehicle_id? There are only 2 that are used. Try both and it should work, also how are you requesting this GET POST? Curl Postman, did you include the right stuff in the headers etc?

It's also a bit hard to understand what you wrote. Wdym you don't know the vehicle id? Is it your car?

**Reply by syt198945** [reply]
Thanks ，IT‘s Done!!!!!!
